sources/packages/web/src/components/common/NoticeOfAssessmentFormView.vue

```diff
@@ -1,52 +1,214 @@
 <template>
-  <formio-container formName="noticeOfAssessment" :formData="initialData" />
+  <formio-container
+    formName="noticeOfAssessment"
+    @customEvent="customEvent"
+    :formData="initialData"
+  />
+  <confirm-modal
+    title="Reissue MSFAA number"
+    ref="confirmReissueMSFAA"
+    okLabel="Reissue MSFAA number now"
+    cancelLabel="Cancel"
+    :loading="msfaaReissueProcessing"
+    ><template #content>
+      <p>
+        <strong>Are you sure you want to proceed?</strong> Reissuing an MSFAA
```

Comment:
Instead of using strong, we can user font-bold

![image](https://github.com/bcgov/SIMS/assets/54600590/624e2b8d-160c-44da-b818-7b40e9635b78)

Reply:
IMO using `<strong>` is semantically better.
```html
<strong>Are you sure you want to proceed?</strong> Reissuing an MSFAA
```
My suggestion would be apply the `font-bold` as a default for the `strong` tag, as below. Does it make sense?
```css
strong {
  @extend .font-bold;
}
```

Reply:
Works for me.

Reply:
Done!